extractor/src/main/java/org/schabi/newpipe/extractor/services/soundcloud/SoundcloudParsingHelper.java

```diff
@@ -189,6 +194,7 @@ public static String resolveUrlWithEmbedPlayer(final String apiUrl) throws IOExc
      *
      * @return the resolved id
      */
+    // TODO: what makes this method different from the others? Don' they all return the same?
```

Comment:
Which method would be the same as which others? They all do different things imo